src/components/cluster/detail/AddNodePool.js

```diff
@@ -0,0 +1,488 @@
+    padding-top: 9px;
+    padding-bottom: 9px;
+  }
+  button {
+    margin-right: 16px;
+  }
+  .availability-zones & {
+    margin-bottom: 22px;
+  }
+  p {
+    margin-left: 15px;
+  }
+`;
+
+const FlexColumnDiv = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: 650px;
+  label {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    margin: 0 0 31px;
+    &.instance-type {
+      margin-bottom: 21px;
+    }
+    p {
+      line-height: 1.4;
+    }
+  }
+  .label-span {
+    color: ${props => props.theme.colors.white1};
+  }
+  .label-span,
+  input,
+  select {
+    font-size: 16px;
+    margin-bottom: 13px;
+    font-weight: 400;
+  }
+  input {
+    box-sizing: border-box;
+    width: 100%;
+    background-color: ${props => props.theme.colors.shade5};
+    padding: 11px 10px;
+    outline: 0;
+    color: ${props => props.theme.colors.whiteInput};
+    border-radius: 4px;
+    border: 1px solid ${props => props.theme.colors.shade6};
+    padding-left: 15px;
+    line-height: normal;
+  }
+  & > input {
+    /* margin-bottom: 13px !important; */
+  }
+  p {
+    margin: 0;
+    font-size: 14px;
+    color: ${props => props.theme.colors.white1};
+  }
+  a {
+    text-decoration: underline;
+  }
+  /* Overrides for AWSInstanceTypeSelector */
+  .textfield label,
+  .textfield,
+  .textfield input {
+    margin: 0;
+  }
+  /* Overrides for NumberPicker */
+  .availability-zones {
+    & > div > div,
+    & > div > div > div {
+      margin: 0;
+    }
+    & div > p {
+      font-size: 16px;
+    }
+  }
+  .scaling-range {
+    form {
+      label {
+        margin-bottom: 7px;
+        color: ${props => props.theme.colors.white1};
+        font-weight: 400;
+      }
+      & > div:nth-of-type(2) {
+        display: none;
+      }
+    }
+  }
+`;
+
+class AddNodePool extends Component {
+  state = {
+    isNameBeingEdited: false,
+    name: '',
+    availabilityZonesPicker: {
+      value: 1,
+      valid: true,
+    },
+    releaseVersion: '',
+    scaling: {
+      automatic: false,
+      min: 3,
+      minValid: true,
+      max: 3,
+      maxValid: true,
+    },
+    submitting: false,
+    valid: false, // Start off invalid now since we're not sure we have a valid release yet, the release endpoint could be malfunctioning.
+    error: false,
+    aws: {
+      instanceType: {
+        valid: true,
+        value: this.props.defaultInstanceType,
+      },
+    },
+    awsInstanceTypes: {},
+    azure: {
+      vmSize: {
+        valid: true,
+        value: this.props.defaultVMSize,
+      },
+    },
+    kvm: {
+      cpuCores: {
+        value: this.props.defaultCPUCores,
+        valid: true,
+      },
+      memorySize: {
+        value: this.props.defaultMemorySize,
+        valid: true,
+      },
+      diskSize: {
+        value: this.props.defaultDiskSize,
+        valid: true,
+      },
+    },
+  };
+
```

Comment:
Do we need kvm and azure? I mean, AFAIK we don't need them now, but we will need them in the future. Then ¿It makes sense to keep them here?